### ISSUE TITLE
Remove none option from banner

### ISSUE THREAD

**Issue by cory-skeers** [uiowa/uiowa#9861]
# Motivation

When no background is set on the banner component, it defaults to black for accessibility. This can be confusing to users since it is represented by the `-None-` option. There is also a `Black` option that renders the same as when `-None-` is set. We should remove the `-None-` option and default to `Black` to reduce confusion.

Megan originally surfaced this issue in Slack:
https://iowaweb.slack.com/archives/C0164FNN1PV/p1775858936443659?thread_ts=1775857948.476719&cid=C0164FNN1PV

# Additional Info

<img width="2419" height="1026" alt="Image" src="https://github.com/user-attachments/assets/f061961e-808e-4c13-8db8-5491e81137d0" />

- Remove `-None-` option
- Default to `Black`
